pkg/cmd/pr/list/list.go

```diff
@@ -54,12 +56,18 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List pull requests in a repository",
-		Long: heredoc.Doc(`
+		// TODO advancedIssueSearchCleanup
+		// Update the links and remove the mention at GHES 3.17 version.
+		Long: heredoc.Docf(`
 			List pull requests in a GitHub repository. By default, this only lists open PRs.
 
+			When runs against %[1]sgithub.com%[1]s or GHE versions newer than 3.17.x, the command
+			performs search by using the advanced issue search syntax, which is documented at:
+			<https://github.blog/changelog/2025-03-06-github-issues-projects-api-support-for-issues-advanced-search-and-more>
+
```

Comment:
Same as other comment above:

```suggestion
			On supported GitHub hosts, advanced issue search syntax can be used in the %[1]s--search%[1]s query. For more information about advanced issue search, see: <https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/filtering-and-searching-issues-and-pull-requests#building-advanced-filters-for-issues>
```

Reply:
Applied.